shopinvader/models/shopinvader_notification.py

```diff
@@ -12,7 +12,7 @@ class ShopinvaderNotification(models.Model):
     _description = "Shopinvader Notification"
 
     backend_id = fields.Many2one(
-        "shopinvader.backend", "Backend", required=True
+        "shopinvader.backend", "Backend", required=False
```

Comment:
@simahawk Why do you make the backend optional?

Reply:
because otherwise you cannot have predefined templates.
The fields is required in the view.